amundsen_application/static/js/config/config-types.ts

```diff
@@ -119,6 +119,7 @@ type SourcesConfig = {
   [id: string]: {
     displayName?: string;
     iconClass?: string;
+    iconPath?: string;
```

Comment:
To get rid of the two new TypeScript errors you got you would need to remove the optionality of displayName and iconPath.

Maybe you don't want to share this type? What if you create a `CodeSource` type that requires iconPath and displayName?

Reply:
I am not sure if we want to do that. from @ttannis review I am re-using existing components. 
From your suggestion, I can do following following so that I do not get type error. 

create new object `DescriptionSourceConfig` similar to `SourceConfig`: 
````
DescriptionSourceConfig =  [id: string]: {
displayName: string;
iconPath: string;
};`

interface TableResourceConfig extends BaseResourceConfig {
  supportedDescriptionSources?: DescriptionSourceConfig;
}
```

@ttannis let me know what do you think about it?

Reply:
That would work too!

Reply:
will make the changes. Thanks!